artifactory_cleanup/artifactorycleanup.py

```diff
@@ -13,6 +15,7 @@ class CleanupSummary:
     policy_name: str
     artifacts_removed: int
     artifacts_size: int
+    removed_artifacts_list: Optional[dict] = None
```

Comment:
```suggestion
    removed_artifacts Optional[dict] = None
```

Is it a `dict?` :thinking: